src/app/_components/MobileNavigation.tsx

```diff
@@ -73,23 +79,17 @@ type NestedMenuItemProps = {
 
 type NestedMenuProps = {
   title: string
-  items: NestedMenuItemProps[]
+  items: Array<NestedMenuItemProps>
   setOpen: Dispatch<SetStateAction<boolean>>
 }
 
 function NestedMenu({ title, items, setOpen }: NestedMenuProps) {
   return (
     <li>
       <span className="mb-4 block text-brand-200">{title}</span>
-      <ul className="space-y-6 border-l">
-        {items.map((item) => (
-          <LinkItem
-            key={item.path}
-            nested
-            label={item.label}
-            path={item.path}
-            setOpen={setOpen}
-          />
+      <ul className="items-center space-y-6 border-l">
+        {items.map((item, i) => (
+          <LinkItem key={i} nested {...item} setOpen={setOpen} />
```

Comment:
@CharlyMartin - Another one - I think O prefer seeing what's passed down..

Reply:
Not sure I have a strong preference for either, it just made sense in the moment :)